registered/parser.py

```diff
@@ -391,13 +449,13 @@ def from_line(cls, parts):
             _,
             route_id,
             pattern_id,
-            description,
-            sequence,
-            _,
-            revenue_type,
-            *_,
+            _trip_type_description,
```

Comment:
question: is there a reason to drop the description, if we know that's what this field is?

Reply:
We don't use it for anything, since we use the trip_type enum instead. I figured there's no reason to take up space saving it. But we can save it if you prefer.